app/webroot/js/wjr/clubsite.js

```diff
@@ -75,21 +75,21 @@ define(['jquery', 'underscore', 'cakebootstrap', 'moment', './forms', './editabl
         }
       }, {
         selector: '.simple-marker-map',
-        requires: ['wjr/map'],
+        requires: ['wjr/map', 'markerclusterer'],
```

Comment:
Can we remove `markerclusterer` here, and for the draggable marker map ? I think you are only using the clusterer for the multi marker map so we can optimize the other page loads (ex: event page) by only loading it when we will use it.

Reply:
Yep, no problem.  Done.